src/EditorFeatures/Core/EditAndContinue/EditAndContinueLanguageService.cs

```diff
@@ -309,7 +371,11 @@ public async ValueTask<bool> HasChangesAsync(string? sourceFilePath, Cancellatio
         }
     }
 
-    public async ValueTask<ManagedHotReloadUpdates> GetUpdatesAsync(CancellationToken cancellationToken)
+    [Obsolete]
+    public ValueTask<ManagedHotReloadUpdates> GetUpdatesAsync(CancellationToken cancellationToken)
+        => GetUpdatesAsync(runningProjects: [], cancellationToken);
```

Comment:
Does this mean things are broken until the debugger respond? Would it be better to pass all something like `currentCompileTimeSolution.Projects.Select(p=>p.FilePath)` as the first parameter?

Reply:
No, shouldn't be broken. The debugger doesn't currently consume the results that are affected by this input.